build/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY . .
 RUN make go-build
 
 ####
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-micro:latest
```

Comment:
What's the reason for this change?

Reply:
From the doc here: https://www.redhat.com/en/blog/introduction-ubi-micro
It has smaller image size, and can reduce the attack surface.